pkg/http/api.go

```diff
@@ -979,3 +992,44 @@ func getMetadataFromRequest(reqCtx *fasthttp.RequestCtx) map[string]string {
 
 	return metadata
 }
+
+func (a *api) onPerformTransaction(reqCtx *fasthttp.RequestCtx) {
```

Comment:
nit: please use `Perform` verb instead of `Perform` here.

Reply:
Renamed to onPostTransaction